source/fuzz/transformation_composite_extract.cpp

```diff
@@ -54,6 +55,12 @@ bool TransformationCompositeExtract::IsApplicable(
   if (!composite_instruction) {
     return false;
   }
+  if (transformation_context.GetFactManager()->IdIsIrrelevant(
+          message_.composite_id())) {
+    // |composite_id| will participate in DataSynonym facts. Thus, it can't be an
+    // irrelevant id.
+    return false;
+  }
```

Comment:
We *could* allow irrelevant ids, and change "Apply" to only add facts when the id is not irrelevant.

Reply:
Fixed in 35f5730.